storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1165,10 +1154,10 @@ bool Redo_Log_Data_Manager::stop_at(lsn_t lsn, lsn_t checkpoint_lsn) {
 
   archived_log_monitor.stop();
 
-  /* check if redo log are disabled */
-  if (!reader.validate_redo_log_file()) {
-    return (false);
-  }
+  /* to ensure redo logs are not disabled during the backup, reopen the log
+  files to read HEADER */
+  reopen_log_files();
+  log_crash_safe_validate(*log_sys);
```

Comment:
if validation fails, we should return error?

Reply:
Fixed by Marcelo in his 8.0.29 Branch